app/assets/stylesheets/uofa_homepage.scss

```diff
@@ -0,0 +1,41 @@
+$splash-content-link: #337ab7;
+$white-text-on-black: #efefef;
+
+.info-pane p > a {
+  text-decoration: none;
+  color: $splash-content-link;
+}
+
+#featured_content img {
```

Comment:
@murny  
Tried removing the ID-based CSS but this one is stubborn. Options as I see them:
* leave as ID-base to override pre-existing selector; aim easy merging future upstream changes by not changing avalon.scss or
* remove pre-existing ID-based selector in avalon.scss so there is no need to override https://github.com/ualbertalib/avalon/blob/feature/theme-mid-372/app/assets/stylesheets/avalon.scss#L462-L464

Reply:
Think this is fine, don't want you to spend a ton of time refactoring this. Key thing is you tried! Could be left as an excercise down the road if we want to tackle this.

Maybe just add a comment in the code above so a ftuure developer that stumbles upon this can get a grasp of why this is, pretty much an explaination of what you just wrote here?

e.g:
```scss
// FIXME: We need to override the styles in avalon.scss here
// should probably fix this and remove the usage of an ID selection
#featured_content img
```

Or something along those lines?